relay-server/src/utils/rate_limits.rs

```diff
@@ -132,6 +132,9 @@ pub struct EnvelopeSummary {
 
     /// Indicates that the envelope contains regular attachments that do not create event payloads.
     pub has_plain_attachments: bool,
+
+    /// Whether the envelope contains a transaction event which already had the metrics extracted.
+    pub metrics_extracted: bool,
```

Comment:
Let's call this `transaction_metrics_extracted`, since session items have their metrics extracted, too.